types/licences.d.ts

```diff
@@ -92,5 +92,34 @@ export interface WarningClient {
   raiseWarning: (bookingId: number, code: string, messsage: string) => Promise<void>
   acknowledgeWarnings: (errorIds: number[]) => Promise<number>
   getOutstandingWarnings: () => Promise<List<Warning>>
-  getAcknowledgedWarnings: () => Promise<List<Warning>>
+  getAcknowledgedWarnings: () => Promise<List<Warning>> 
+}
+
+interface ProbationAreas {
+  code: string;
+  description: string;
+}
+
+interface Ldus {
```

Comment:
both `ProbationAreas` and `Ldus` should be singular, as they only represent one item

Reply:
done